### PR TITLE
Add ProfileInfo for ProfilePage / Add basic authorization logic

### PR DESCRIPTION
**Pull request by MaksimkaSota**
1. Fixed data display error (useMounted);
2. Added one of logic ProfileInfo (move from UsersPage to ProfilePage using useParams);
3. Added basic authorization logic (Cookie);

In the future I will reduce the code using thunk creators and server requests in separate files!
Server errors will be handled at a later stage of development!

### CODE REVIEW

File: src/Components/Main/Pages/ProfilePage/ProfileInfo/ProfileInfo.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import classes from './ProfileInfo.module.scss';
+import userPhoto from '../../../../../assets/images/user.png';
+import { Preloader } from '../../../../Common/Preloader/Preloader';
+import { useMounted } from '../../../../../hooks/useMounted';
+
+export const ProfileInfo = ({profile, isFetchingProfile}) => {
+  const mounted = useMounted();
```
Comment (ZhekaVasil) [inline src/Components/Main/Pages/ProfilePage/ProfileInfo/ProfileInfo.js]: Seems like mounted hook should not be in the component
The component just takes props and render them, so it is simple component which can be rendered without issues when provided props are correct
I would suggest to add mounted hook in parent component like:
```
{mounted && <ProfileInfo />}
```

File: src/Components/Main/Pages/UsersPage/UsersPage.js [hunk 2]
```diff
@@ -13,32 +14,36 @@ export const UsersPage = ({
                             setPage,
                             setUsers,
                             setTotalCount,
-                            isFetching,
-                            toggleIsFetching
+                            isFetchingUsers,
+                            toggleIsFetchingUsers
                           }) => {
+  const mounted = useMounted();
```
Comment (ZhekaVasil) [inline src/Components/Main/Pages/UsersPage/UsersPage.js]: The same applies here =)
It makes more sense to add mounted hook into parent component

File: src/Components/Main/Pages/ProfilePage/ProfileInfo/ProfileInfo.js (hunk 1 above)
Comment (MaksimkaSota) [reply src/Components/Main/Pages/ProfilePage/ProfileInfo/ProfileInfo.js]: This is a very good idea, thanks!
Done!

File: src/Components/Main/Pages/UsersPage/UsersPage.js (hunk 2 above)
Comment (MaksimkaSota) [reply src/Components/Main/Pages/UsersPage/UsersPage.js]: Done!

**Review by ZhekaVasil — APPROVED** [review]

**Pull request merged**